tests/Unit/DataStructures/DataBox/TestHelpers.hpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#pragma once
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+namespace test {
```

Comment:
why not `TestHelpers`?

Reply:
`test` is shorter 😛 I'm fine with `TestHelpers`, but then we should document it...

Reply:
We have been using `TestHelpers` and did tell Will to move something else into a `TestHelpers` namespace :P